docs/README.md

```diff
@@ -301,7 +301,35 @@ Supported events:
 ### Advanced Logic
 Validators can be grouped together with `AND` and `OR` operators:
 
-TODO: Show examples.
+```yml
+- do: description
+  and:
+    - must_include:
+        regex: 'Test Plan'
+        message: 'Test plan must be included'
+    - must_include:
+        regex: 'Goal'
+        message: 'Please include the goal of the PR'                 
+```
+
+`AND` and `OR` operators can also be nested
+
+```yml
+- do: label
+ or:
```

Comment:
This looks like the wrong indentation. However, it may not be wrong yml -- just does not line up when compared to the example above.